nms/server/auth/util.js

```diff
@@ -18,13 +18,13 @@ import type {FBCNMSMiddleWareRequest} from '../../server/middleware';
 import type {UserRawType} from '../../shared/sequelize_models/models/user';
 import type {UserType} from '../../shared/sequelize_models/models/user.js';
 
-import EmailValidator from 'email-validator';
```

Comment:
The linter with its new TypeScript power understands that  `'email-validator'` does not have a default export so this needs to fixed.